### ISSUE TITLE
Table filter menu does not work on mobile, closes virtual menu

### ISSUE THREAD

**Issue by Th14go** [primefaces/primeng#11532]
**Current behavior**
When using the table filter menu on mobile, it closes the virtual keyboard

**Expected behavior**
The correct behavior was to keep the virtual keyboard open

**Minimal reproduction of the problem with instructions**
Using the table without responsive mode we were able to check the bug in the table, clicking on the filter quickly opens and closes the virtual keyboard.

https://stackblitz.com/edit/primeng-tablefilter-demo-ltcvgh?file=src/app/app.component.ts

* **Angular version:**
13.2.3
* **PrimeNG version:** 5.X
13.3.3
* **Browser:** 
Chrome 101X
**Please tell us about your environment:**
Ubuntu 22.04

**Comment by cetincakiroglu** [comment]
Hi,

I couldn't replicate the problem. Could you please give some instructions?

Regards

**Comment by Th14go** [comment]
Hi, to replicate the problem open the stackblitz link on the mobile and click on preview, you will see that clicking on the filter opens the keyboard, right after it closes, then it is not possible to do the search.

you can check this link I recorded the screen to replicate the bug:
https://1drv.ms/v/s!AkUtqKpFOWwGguw3PHF_ZeyDuSlVPQ

**Comment by mertsincan** [comment]
Duplicate of [11567](https://github.com/primefaces/primeng/issues/11567)

**Comment by Th14go** [comment]
The problem was solved ?

**Comment by mertsincan** [comment]
Yes, fixed with https://github.com/primefaces/primeng/commit/2908cf344c679bf9497cc04e5a74ecb87f0ef4a5#diff-54f9fcc81f3c7d880cfe5f7ddd5dda1e60f15581a7ce88dd922b0586b1f259a1R4797

**Comment by Th14go** [comment]
@mertsincan  wonder !, Thanks, how do i get this fix, i'm on primeng 13.3.3 ?

**Comment by frauscher** [comment]
@Th14go you can implement an own directive and override the default behaviour:
```
@Directive({
  selector: '[customDirective]'
})
export class CustomDirective {
  constructor(@Host() @Self() @Optional() public hostOverlayPanel: OverlayPanel) {
    // Now you can access specific instance members of host directive
    let app = (<any>hostOverlayPanel)._app;

    // Override the onWindowResize method.
    hostOverlayPanel.onWindowResize = () => {
      if (!DomHandler.isAndroid()) {
        hostOverlayPanel.hide();
      }
    };
  }
}
```

**Comment by mertsincan** [comment]
> @mertsincan wonder !, Thanks, how do i get this fix, i'm on primeng 13.3.3 ?
Unfortunately, You need to wait for the next release.

**Comment by developeruptown** [comment]
> @mertsincan wonder !, Thanks, how do i get this fix, i'm on primeng 13.3.3 ?

You sure can do it, 
here is how you do it.
1)You Fork Primeng
2) Make appropriate changes 
3) Remove Primeng from your package.
4) install package using your (updated)forked package to your project.
here are more details.
Step 1 and 2
https://stackoverflow.com/questions/53879891/replacing-file-from-node-modules
Step 3 and 4
https://stackoverflow.com/questions/40528053/npm-install-and-build-of-forked-github-repo
